Automattic/wp-calypso
# Sites Management Page: Create hook to allow scrolling to top of window or an element

#### Proposed Changes

see [comment](pdtkmj-nh-p2#comment-642)

* Add a hook `useScrollToTop` to handle scroll to top logic
* Add scroll to top button to sites dashboard
* Refactor help center back to top button to use the `useScrollToTop` to separate logic from UI
*
![image](https://user-images.githubusercontent.com/47489215/186783814-85f9b30a-f9e6-4871-9874-f409a7a62913.png)


#### Testing Instructions



* Go to `/sites`
* Scroll down list of sites, you should see a button pops up after the scroll threshold that allows scrolling to the top.

**_Note I have perform regression on help center back to top button and it works as expected._** help center is currently disabled globally, to test:

- set help-center to true in `config/development.json`
- add `require_once __DIR__ . '/help-center/class-help-center.php'; ` to `apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php` method `load-help-center`
- sync changes to sandbox with `yarn dev --sync` from `apps/editing-toolkit` dir
- on FSE theme go to editor and browse an article in help center, verify back to top button works.
![image](https://user-images.githubusercontent.com/47489215/187315312-4a9342a1-21d4-4c2d-a060-ab5be43b6909.png)
(nevermind the word toppy 😄 )


#### Pre-merge Checklist






Related to #
